spanner/api/Spanner/Program.cs

```diff
@@ -1888,6 +1943,267 @@ await cmd.ExecuteReaderAsync().ConfigureAwait(false))
             }
         }
 
+        public static object InsertStructSampleData(string projectId,
+            string instanceId, string databaseId)
+        {
+            var response = InsertStructSampleDataAsync(
+                projectId, instanceId, databaseId);
+            s_logger.Info("Waiting for operation to complete...");
+            response.Wait();
+            s_logger.Info($"Response status: {response.Status}");
+            return ExitCode.Success;
+        }
+
+        public static async Task InsertStructSampleDataAsync(
+            string projectId, string instanceId, string databaseId)
+        {
+            // [START spanner_write_data_for_struct_queries]
+            string connectionString =
+            $"Data Source=projects/{projectId}/instances/{instanceId}"
+            + $"/databases/{databaseId}";
```

Comment:
I'd prefer it to be as you suggest too. Since this is code to be included in documentation (which has width limitations) and it mirrors the existing approach already in the sample. I'm going to opt to leave as it for this PR.